packages/ebayui-core/src/components/ebay-table/examples/frozen-header.marko

```diff
@@ -0,0 +1,46 @@
+import data from "./data.json";
+
+<ebay-table style={height: "300px"} frozen-header ...input>
```

Comment:
The inline style in Skin was only to allow the frozen header to be demonstrated, so we should remove it from here.

While we're at it, can you remove it from the example code in Skin?

Reply:
Ah...it looks like I missed the max-height in Skin. 

<kbd><img width="1073" alt="image" src="https://github.com/user-attachments/assets/0a21b6aa-069a-42df-bc4f-5fa717e19555" /></kbd>

Reply:
I made the Skin fixes, so these can be removed.